SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDKFunctionalTests.Build.cs

```diff
@@ -10,6 +10,8 @@ public SpatialGDKFunctionalTests(ReadOnlyTargetRules Target) : base(Target)
         PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
         bUseUnity = false;
 
+        PublicIncludePaths.Add(ModuleDirectory);
```

Comment:
What includes do we need on this path?

Reply:
With this path, we can do `#include "SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.h"` as opposed to `#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.h"` - I don't feel like include paths inside modules should use the name of the module inside of it (even though we have `MODULENAME_API`...)
Not that I have a strong opinion regarding this :)

Reply:
Yeah fair enough. I'm not 100% on why we didn't put these files in the usual private/public breakdown, but I guess whats done is done. Does seem like we could just add this to the PrivateIncludesPaths though.